src/app/visual/summary-action-bar/summary-action-bar-demo.component.html

```diff
@@ -81,5 +85,61 @@
         <sky-key-info-label>Field 3</sky-key-info-label>
       </sky-key-info>
     </div>
+    <div *skyThemeIf="'modern'">
```

Comment:
Is the only difference here the `layout`? I know that we use the `skyThemeIf` for most things - but would it just be better to grab the theme and conditional the `layout` on these? It seems like a lot of extra repeated code just for `layout`.

Reply:
Yep. Updated.